ivy/functional/frontends/paddle/tensor/manipulation.py

```diff
@@ -171,5 +186,6 @@ def take_along_axis(arr, indices, axis):
     "paddle",
 )
 @to_ivy_arrays_and_back
-def rot90(x, k=1, axes=(0, 1), name=None):
-    return ivy.rot90(x, k=k, axes=axes)
+def moveaxis(x, source, destination, name=None):
```

Comment:
you'll need to add the unsupported dtype decorator etc for this function as well